# Optimal path from MPPI optimization
std_msgs/Header header
geometry_msgs/PoseStamped[] path_points
float64[] costs
float64 total_cost